tests/functional/report_viewer_api/test_hash_clash.py

```diff
@@ -160,8 +160,8 @@ def test_hash_clash(self):
                                                  run_id1,
                                                  'XXX',
                                                  ((2, 1), (2, 2)))
-            # Same file, same hash and different position
-            self.assertNotEqual(rep_id1, rep_id3)
+            # Same file, same hash and different position in line
```

Comment:
`assertNotEqual` has been changed to `assertEqual` because as of this commit we consider two bug paths the same if only the line numbers of the path event change (e.g. the definition of a function got few lines down). See the changes here: https://github.com/Ericsson/codechecker/pull/764/files#diff-b2e3db7161aa24606040a31602cb60a5L1626. By the way this is considered a heuristic, since we don't check if the line numbers in a bug path move equally.